Gaijinizer patch/Map240.txt

```diff
@@ -14,60 +14,65 @@ it.#MTLed
 
 普段はあんな場所までいかないからね……
 > CONTEXT: Map240/events/1/pages/0/15/Dialogue/kazuya
-Yeah......... well, indeed. It was really like
-a walk but...I usually do not go to such a
-place ...#MTLed
+Yeah, I suppose it was kind of like an actual
+walk, but we don't normally go to those kinds of
+places...#RonIffy
 > END STRING
 
 > BEGIN STRING
 …………ええ？
 
 またあのお店の焼き鳥食べたいって……？
 > CONTEXT: Map240/events/1/pages/0/24/Dialogue/kazuya
-...... yeah? Would you like to eat the grilled
-chicken at that shop again ...?#MTLed
+...Hmm?
+
+You want to go eat some grilled chicken at that
+shop again...?#Ron
 > END STRING
 
 > BEGIN STRING
 あそこ、美味しいけど高いんだよね……
 
 ニーナも大好きだけど……
 > CONTEXT: Map240/events/1/pages/0/28/Dialogue/kazuya
-Over there, it's delicious but expensive ...
-I also love Nina ...#MTLed
+That place is good but it's pretty expensive...
+
+Nina loves it too...#Ron
 > END STRING
 
 > BEGIN STRING
 キャウン！キャウン！
 > CONTEXT: Map240/events/1/pages/0/32/Dialogue/バーゲスト
-Caw! Caw!#MTLed
+Wan! Wan!#CheckLater
 > END STRING
```

Comment:
Note

Reply:
I just didn't know what the onomatopoeia was supposed to be, I looked it up and found nothing. I'm assuming some kind of high-pitched bark of happiness.

Reply:
It's fine like this if you want to keep it like this.